pages/vi/vi-planetapps.md

```diff
@@ -57,7 +57,7 @@ As you can see from the image below, there is an upgrade ready to be downloaded.
 
 ![Fetch Items from the nation](images/vi-planet-fetch-items.png "Manager Dashboard in your localhost")
 
-1. Click the `Upgrade` button on Manager page and it take you to upgrade page. An upgrade refers to a software update which improves the Planet (Note: If you do not see `Upgrade` button then simply carry on). It will then ask you for verification.
+1. Click the `Upgrade` button on Manager page and it take will you to upgrade page. An upgrade refers to a software update which improves the Planet (Note: If you do not see `Upgrade` button then simply carry on). It will then ask you for verification.
```

Comment:
please fix this

Reply:
@lmmrssa I have added a second commit to fix the issue you pointed. Thank you.